raspios/overlay-rootfs/root/start_chromium.sh

```diff
@@ -1,12 +1,18 @@
 #!/bin/bash
 
-/usr/bin/chromium-browser http://localhost:808ROBOT_ID \
+/usr/bin/chromium-browser http://localhost:8080 \
```

Comment:
I would rename the commit title `raspios: disable chromium cache` by `raspios: disable unused features`

Reply:
Sorry forgot a word:
`raspios: disable unused chromium features`

Reply:
The purpose this commit is really to disable Chromium cache. Other options are just optimisations.
I changed the title into `raspios: disable chromium cache and unwanted features`.